core/app/modules/import-export/module.php

```diff
@@ -0,0 +1,120 @@
+<?php
+namespace Elementor\Core\App\Modules\ImportExport;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Common\Modules\Ajax\Module as Ajax;
+use Elementor\Plugin;
+use Elementor\Settings;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Import Export Module
+ *
+ * Responsible for initializing Elementor App functionality
+ */
+class Module extends BaseModule {
+	const FORMAT_VERSION = '1.0';
+
+	const EXPORT_TRIGGER_KEY = 'elementor_export_kit';
+
+	const IMPORT_TRIGGER_KEY = 'elementor_import_kit';
+
+	/**
+	 * @var Export
+	 */
+	private $export;
+
+	/**
+	 * @var Import
+	 */
+	private $import;
+
+	/**
+	 * Get name.
+	 *
+	 * @access public
+	 *
+	 * @return string
+	 */
+	public function get_name() {
+		return 'import-export';
+	}
+
+	public function get_init_settings() {
+		$export_nonce = wp_create_nonce( 'elementor_export' );
+		$export_url = add_query_arg( [ 'nonce' => $export_nonce ], Plugin::$instance->app->get_base_url() );
+
+		return [
+			'exportURL' => $export_url,
+		];
+	}
+
+	private function on_elementor_init() {
+		if ( isset( $_POST['action'] ) && self::IMPORT_TRIGGER_KEY === $_POST['action'] ) {
```

Comment:
Check permissions (the admin menu is not enough)

Reply:
Added a validation before loading the module